src/component/cyfs-group/src/storage/engine/storage_engine_group_state.rs

```diff
@@ -506,7 +506,7 @@ impl StorageEngineGroupStateWriter {
             assert!(is_new);
         }
         let adding_set_id = add_single_op_env.commit().await?;
-        let prev_value = self
+        let _prev_value = self
```

Comment:
It would be more appropriate to change this line to the following form
```
self.op_env.set_with_path(self.state_path.adding(), &adding_set_id, &None, true).await?;
```